### PR TITLE
Make the fileviewer fill the container which it is inside

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**
Instead of filling the remaining amount of the screen we just fill whichever container it was placed inside

### CODE REVIEW

File: fileViewer.scss [hunk 1]
```diff
@@ -7,5 +7,5 @@
 	@import './src/plugins/html/html.scss';
 	@import './src/plugins/image/image.scss';
 	@import './src/plugins/pdf/pdf.scss';
-
+	height: 100%;
```
Comment (dlockhart) [inline fileViewer.scss]: Do you need this? Typically `height: 100%` doesn't do anything...

File: src/plugins/html/html.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
 .vui-fileviewer-html-native {
 	width: 100%;
+	height: 100%;
```
Comment (dlockhart) [inline src/plugins/html/html.scss]: Same here.

File: fileViewer.scss (hunk 1 above)
Comment (jstefaniuk-d2l) [inline fileViewer.scss]: Without this one the iframe didn't resize to the full amount, I'll try removing the styling on the viewer ones though

Comment (jstefaniuk-d2l) [inline fileViewer.scss]: Just looked into it, without the 'height:100%' the iframe doesn't size to the full span of it's container

Comment (dlockhart) [inline fileViewer.scss]: OK cool, are the other ones necessary?

Comment (jstefaniuk-d2l) [inline fileViewer.scss]: Yep

**Pull request merged**